src/main/java/org/folio/rest/impl/InvoicesImpl.java

```diff
@@ -30,6 +30,11 @@ public void postInvoiceInvoices(String lang, Invoice invoice, Map<String, String
                                   Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     InvoiceHelper helper = new InvoiceHelper(okapiHeaders, vertxContext, lang);
 
+    if (!helper.validateNewInvoice(invoice)) {
```

Comment:
I think that this check is also needed when updating an invoice.

Reply:
Done